odata-data-aggregation-ext/5 Vocabulary for Data Aggregation.md

```diff
@@ -2,7 +2,7 @@
 
 # ##sec Vocabulary for Data Aggregation
 
-The following terms are defined in the vocabulary for data aggregation [OData-VocAggr](#ODataVocAggr).
+The following terms are defined in the vocabulary for data aggregation [OData-VocAggr](#ODataVocAggr) together with the `UpNode` instance annotation introduced in [section ##Transformationtraverse].
```

Comment:
Curiosity kils the 😹, but: Does this `[section ##Transformationtraverse]` reference works for the html and pdf output channels as OASIS admins render the markdown?

Reply:
PDF generation isn't decided yet, any hints for good tools for converting HTML to PDF are welcome.

Current approach would be to use FireFox' Print feature, but this has problems with hyperlinks:
* Using "Save as PDF" produces a nice and slender PDF, but all hyperlinks to sections actually target the HTML version.
* Using "Microsoft Print to PDF" produces a rather large PDF with none of the hyperlinks working, although they are formatted as hyperlinks.

Reply:
"Save as PDF" in Google Chrome produces correct internal hyperlinks targeting the PDF document itself.

Reply:
Now we only have to figure out how to adjust the header and footer to match the OASIS standard ones - this was why we planned to use Firefox/Waterfox.

Reply:
This may be the way forward, see first two answers to https://stackoverflow.com/questions/44575628/alter-the-default-header-footer-when-printing-to-pdf

Reply:
Or use headless Chrome via Puppeteer, see #6.